spec/support/helpers.rb

```diff
@@ -8,5 +8,13 @@ def with_isolated_config
     RSpec.configuration = original_config
   end
 
+  def type_metatag(type)
+    if RUBY_VERSION > '1.8.7'
+      "type: :#{type}"
+    else
+      ":type => :#{type}"
+    end
+  end
```

Comment:
This should only be defined once. We have a place for similar checks: https://github.com/rspec/rspec-rails/blob/master/lib/rspec/rails/feature_check.rb